extensions/blocks/slideshow/style.scss

```diff
@@ -236,3 +236,39 @@
 		}
 	}
 }
+
+@media only email {
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_container {
+		overflow: visible;
+		opacity: 1;
+		width: auto;
+		height: auto;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_container ul.wp-block-jetpack-slideshow_swiper-wrapper,
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_slide figure {
+		display: block;
+		margin-bottom: 12px;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_container ul.wp-block-jetpack-slideshow_swiper-wrapper,
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_slide {
+		list-style: none;
+		margin-left: auto;
+		margin-right: auto;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_slide {
+		display: inline-block;
+		width: 42%;
+		margin-left: 2% !important;
+		margin-right: 2% !important;
+		height: auto;
+		vertical-align: top;
+	}
+	.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_caption.gallery-caption {
+		position: relative;
```

Comment:
While fixing the captions on D52009-code I noted that `position` is [unsupported on many email clients](https://www.campaignmonitor.com/css/positioning-display/position/), so I wonder if this is needed? Gmail at least completely ignores this, and the caption seems to be in the right place on my tests.

Reply:
Aha, at one point during testing I was being too overzealous with my overrides because things didn't seem to be working. You're right, this probably unnecessary.

Reply:
I take that back, it's necessary. 😆 Apple Mail shows up like this unless we override the absolute positioning on the captions:
<img width="614" alt="Screen Shot 2020-10-30 at 10 21 50 AM" src="https://user-images.githubusercontent.com/2124984/97716290-cefdf380-1a99-11eb-9970-3451227e6e9b.png">

So I'll add it back to be safe!